src/libraries/Common/src/Interop/Unix/Interop.Libraries.cs

```diff
@@ -7,12 +7,12 @@ internal static partial class Interop
     internal static partial class Libraries
     {
         // Shims
-        internal const string SystemNative = "System.Native";
-        internal const string GlobalizationNative = "System.Globalization.Native";
-        internal const string NetSecurityNative = "System.Net.Security.Native";
-        internal const string CryptoNative = "System.Security.Cryptography.Native.OpenSsl";
-        internal const string CompressionNative = "System.IO.Compression.Native";
-        internal const string IOPortsNative = "System.IO.Ports.Native";
+        internal const string SystemNative = "libSystem.Native";
```

Comment:
would `lib.System.Native` (and so forth) be better? more readable than `libSystem.Native` perhaps

Reply:
no, this is not the Linux convention. usually `lib' is prefixed without dots.

Reply:
plus adding the `.` might make it such that the loader won't find the library automatically as for anybody PInvoking to System.Native, I believe the runtime will try either System.Native or preappending lib, but it won't find it if it also needs to pre-append a dot.

Reply:
Makes sense